drivers/openstack/openstack.go

```diff
@@ -160,6 +161,11 @@ func GetCreateFlags() []cli.Flag {
 			Usage: "OpenStack floating IP pool to get an IP from to assign to the instance",
 			Value: "",
 		},
+		cli.IntFlag{
+			Name:  "openstack-ip-version",
```

Comment:
@dmp42 Yes, as discussed in an earlier issue, every flag should get a corresponding env variable. It is especially useful for integration tests. It can be done here, otherwise i'll do it in https://github.com/docker/machine/pull/1878 once this one will be merged.